scipioninstaller/installer.py

```diff
@@ -301,6 +301,8 @@ def main():
         # Check Scipion home folder and create it if apply.
         solveScipionHome(scipionHome, dry, noAsk)
         scipionEnv = args.n
+        if not conda:
```

Comment:
I'd check if scipionEnv ==SCIPION_ENV too

Reply:
Me too and I would append the suffix 'env' to the environ name, resulting in `.scipion3env` in order to avoid having something like (only if virtualenv is used and no environ-name is passed):
```
scipion3         # SCIPION_HOME usually is scipion3
  ├── .scipion3  # virtualenv environ
  ├── scipion3   # Scipion launcher
(...)
  └──  whatever
```